src/js/actions/layers.js

```diff
@@ -1563,28 +1563,29 @@ define(function (require, exports) {
      * @returns {Promise}
      */
     var setLocking = function (document, layer, locked) {
-        var payload = {
+        if (layer.isBackground) {
+            return _unlockBackgroundLayer.call(this, document, layer);
```

Comment:
Note to @pineapplespatula: here's another opportunity for a private-action conversion.